### PR TITLE
Add Director of Business Operations

### PR DESCRIPTION
**Pull request by dadlerj**

### CODE REVIEW

File: handbook/ops/roles/director_business_operations.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+1. We give you a small case study project to see how you would approach the job at Sourcegraph.
+1. We schedule a few more hours of interviews.
+1. We check your references.
+1. We make you a job offer.
+
+We want to ensure Sourcegraph is an environment that suits your working style and empowers you to do your best work, so we are eager to answer any questions that you have about us at any point in the interview process.
+
+If you're interested in Sourcegraph but aren't yet ready to apply we are still happy to connect and answer any questions that you might have: [DM us on Twitter](https://twitter.com/srcgraph) or email hiring@sourcegraph.com.
+
+**[Click here to apply]()**
```
Comment (dadlerj) [inline handbook/ops/roles/director_business_operations.md:64]:
```suggestion
**[Click here to apply](https://jobs.lever.co/sourcegraph/4d8ff305-71c8-4b28-a328-461c58819053/apply)**
```

File: handbook/ops/roles/director_business_operations.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](https://about.sourcegraph.com/company/remote) that is [distributed across the world](https://about.sourcegraph.com/company/team). Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+We provide [competitive compensation](https://about.sourcegraph.com/handbook/people-ops/compensation) and [practical benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy and healthy so that you can do your best work.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](https://about.sourcegraph.com/handbook/).
+
+## Interview process
+
+1. You [apply here]().
```
Comment (dadlerj) [inline handbook/ops/roles/director_business_operations.md:53]:
```suggestion
1. You [apply here](https://jobs.lever.co/sourcegraph/4d8ff305-71c8-4b28-a328-461c58819053/apply).
```

File: handbook/ops/roles/director_business_operations.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
+# Director of Business Operations
+
+Sourcegraph's business operations team empowers our product, design, and engineering organizations with the insights, tools, and data they need to make better decisions. As the Director of Business Operations, you will build the team that helps our product organization scale and serve every developer in the world.
```
Comment (ebrodymoore) [inline handbook/ops/roles/director_business_operations.md:3]: I think we should significantly broaden the scope of BizOps. It looks to be based off my current focus and our job openings but in reality I see product/eng support as a sub-team within BizOps or Product (where growth teams are sometimes located) and BizOps as a much broader function.

Given that, I would recommend we keep it much more broad so we cater to candidates who can build out teams that are beyond data infrastructure and product support. For example, Notion's BizOps team is:

> The mission of the Business Operations and Finance team is to bring data insight, operational capability and business expertise to promote strong execution. We partner with the executive team to optimize company-wide resource allocation by defining business strategies, driving cross-functional projects, and unlock new growth opportunities. Our team provides strategic perspectives and analytical horsepower to solve critical business problems (such as GTM and pricing strategy) across the organization."

File: handbook/ops/roles/director_business_operations.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+- Conduct a comprehensive review of our business operations, including the metrics and goals we track, the data we collect, to systems we use, and the teams and projects we serve, and build a team growth roadmap.
+  - Conduct interviews with the entire executive team to understand gaps and needs.
+- Work closely with and get to know our existing business operations and broader ops team.
+- Go through an intensive training program on our product, our value propositions, our customers, our team, and our processes.
+- Take ownership of our internal and external reporting and our business operations team goals.
+
+
+### In the first 60 days you will…
+
+- Begin to build a high-agency business operations team, likely including new hires such as:
```
Comment (ebrodymoore) [inline handbook/ops/roles/director_business_operations.md:22]: Do you expect this hire to settle in before hiring for the other positions, or we'll hire the positions concurrently?

File: handbook/ops/roles/director_business_operations.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  - Business Operations Analyst
+  - Data Engineer
+  - Project Manager
+- Take hands-on ownership of 1–2 large-scale projects to improve or reshape our business operations playbook, or to establish new reporting for teams that need it.
+- Establish plans for and document a new cross-functional Project Management and internal consulting capability.
+- Define new goals for the business operations team.
+
+### In the first 90 days you will…
+
+- Continue to build and scale our business operations capability in line with our Product and Engineering team growth.
```
Comment (ebrodymoore) [inline handbook/ops/roles/director_business_operations.md:32]: Same comment as above about broadening the scope of BizOps

**Review by ebrodymoore — CHANGES_REQUESTED** [review]

File: handbook/ops/roles/director_business_operations.md (hunk 3 above)
Comment (dadlerj) [reply handbook/ops/roles/director_business_operations.md:3]: Thank you for this! I think we agree on the substance here, so getting a review on my wording is really helpful. Let me think through this wording and I'll propose an update.

File: handbook/ops/roles/director_business_operations.md (hunk 4 above)
Comment (dadlerj) [reply handbook/ops/roles/director_business_operations.md:22]: Concurrently (for certain roles). This is written with real-world, current needs, and if it changes, we would update.

**Review by ebrodymoore — APPROVED** [review]
Looks great!

**Pull request merged**